src/test/java/org/folio/fqm/service/FqlToSqlConverterServiceIT.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.fqm.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.folio.fqm.IntegrationTestBase;
+import org.folio.querytool.domain.dto.EntityType;
+import org.folio.querytool.domain.dto.EntityTypeColumn;
+import org.folio.querytool.domain.dto.RangedUUIDType;
+import org.folio.querytool.domain.dto.StringType;
+import org.junit.jupiter.api.Test;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+@Testcontainers
+class FqlToSqlConverterServiceIT extends IntegrationTestBase {
+
+  @Test
+  void valueFunctionTest() throws JsonProcessingException {
```

Comment:
I originally had this split between a few methods, but so much of it is specific to this test that I figured it's okay to keep as one for now. We can always split it up later when we add more tests, since then it will be more clear how we should do that